stream-analytics-queries/production-job-delayed/production-job-delayed.asaql

```diff
@@ -87,14 +86,13 @@ SELECT
   jobId AS jobId,
   sensorId AS sensorId,
   producedQuantity AS actualQty,
-  CASE
-    WHEN expectedQuantity > requestedQuantity THEN requestedQuantity
-    ELSE expectedQuantity
-  END AS expectedQty,
+  expectedQuantity AS expectedQty,
   eventEnqueuedUtcTime AS timestamp,
   eventEnqueuedUtcTime AS sensorTimestamp,
   System.Timestamp AS processingTimestamp,
-  'ProductionJobDelayed' AS notificationType
+  'ProductionJobDelayed' AS notificationType,
+  CONCAT('ProductionJobDelayed:ActualQuantity:', jobId) AS publishedMetric,
+  'Part Out' AS publishedMetricDisplayName
```

Comment:
Don't we want to have "Production Delayed" or something like this here? Similar to the one in Production quality.

Reply:
This is how it is in V1.
I need to investigate and a play a bit more with this column to understand its impact.
But for now i am just aligning with v1 and i will circle back.

Let's ignore this for now.